libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/Certs.hs

```diff
@@ -241,7 +244,7 @@ certsPreds UnivSize {usMinCerts, usMaxCerts} p = case whichTxCert p of
           ( 1
           , sRegPool ^$ poolParams
           ,
-            [ -- Pick a random PoolParams, except constrain the fields poolId, poolRewAcnt, poolOwners, and pooMetadaa
+            [ -- Pick a random PoolParams, except constrain the fields poolId, poolRewAcnt, poolOwners, and poolMetadata
```

Comment:
Previous name was funnier :smile:

Reply:
I could rename the field instead :thinking: